src/main/kotlin/docs/11_Advanced_Topics/C06_Compute_shaders.kt

```diff
@@ -15,54 +15,81 @@ shader support only works on systems that support OpenGL 4.3 or higher. This exc
 """
 
     @Text "## Example use"
-    @Text """Here we provide a quick break-down on compute shader use. This text is preliminary and will be improved
-shortly."""
+    @Text """
+This example is composed of two code blocks. The first block is a compute 
+shader program written in GLSL which produces an `outputImg` by mixing three input 
+colors:
+* A pixel sampled from `inputImg`.
+* A `fillColor` sent as a uniform from Kotlin.
+* A color generated by calculating the cosine of the coordinates of the pixel 
+curretly being processed.
+
+"""
 
 
-    @Text "In the example we will work with the following compute shader in `data/compute-shaders/fill.cs`."
+    @Text "A typical location for such a compute shader could be `data/compute-shaders/fill.cs`."
     @Text """```glsl
 #version 430
 layout(local_size_x = 1, local_size_y = 1) in;
-uniform writeonly image2D outputImg;
-uniform vec4 fill;
 
+uniform vec4 fillColor;
+uniform float seconds;
 layout(rgba8) uniform readonly image2D inputImg;
+uniform writeonly image2D outputImg;
+
 void main() {
     ivec2 coords = ivec2(gl_GlobalInvocationID.xy);
-    float v = cos(coords.x * 0.01 + coords.y * 0.01) * 0.5 + 0.5;
+    float v = cos(coords.x * 0.01 + coords.y * 0.01 + seconds) * 0.5 + 0.5;
+    vec4 wave = vec4(v, 0.0, 0.0, 1.0);
+    vec4 inputImagePixel = imageLoad(inputImg, coords);
 
-    vec4 inxel = imageLoad(inputImg, coords);
-    vec4 pixel = vec4(v, 0.0, 0.0, 1.0);
-    imageStore(outputImg, coords, pixel+inxel+fill);
-}        
+    imageStore(outputImg, coords, wave + inputImagePixel + fillColor);
+}
 ```"""
 
     @Media.Image
     """
     media/compute-shaders-001.png
     """
 
+    @Text """
+The second code block is an OPENRNDR program making use of the compute shader:
+1. It creates a compute shader program from a file. 
+1. Initializes the input buffer using a image loaded from disk.
+1. Inside the `extend` block (called multiple times per second) some uniforms 
+are updated: a `fillColor`, the time in 
+`seconds`, the `inputImg` and the `outputImg`.  In this example only 
+the time changes on every frame.
+1. The compute shader is executed, writing the result to the 
+`outputBuffer`
+1. Finally, the result is displayed by calling `drawer.image()`
```

Comment:
Do you want to punctuate the items in the list or not? However you answer, make sure you keep it consistent across lists. The last 2 items stand out here for not having full stops.